lib/dayron/adapter.ex

```diff
@@ -15,13 +15,12 @@ defmodule Dayron.Adapter do
         end
       end
   """
-  require HTTPoison
```

Comment:
@flaviogranero I am not sure why this changes appear here if I already merge #35